scripts/showcase/README.md

```diff
@@ -0,0 +1,29 @@
+# Showcase Integration Testing
+
+## Requirements
+
+* Install `go` in your `PATH`
+
+## Scripts
+
+* `install.sh <version>`
```

Comment:
This could be subjective, shell scripts are useful for encapsulating complex logics, but maybe these scripts are simple enough so that we don't have to wrap them up. We may also want to expose more functionality of `gapic-showcase` command since sometimes we may want to test directly, wrapping it up will kind of hide it from developers.

Reply:
Understood. For now, I'm going to leave them in just for my own ease of iterating -- but we can remove them or rework them as we get closer to the merge.

Reply:
Resolved by moving scripts into `DEVELOPMENT.md`